include/core/manifolds.h

```diff
@@ -97,6 +97,40 @@ class BoundaryPostprocessor : public DataPostprocessorScalar<dim>
   {
     auto current_cell = input_data.template get_cell<dim>();
 
```

Comment:
I actually added the virtual function `evaluate_scalar_field` because only `evaluate_vector_field` was implemented in Lethe. 
The  `evaluate_vector_field` function do not work for the dummy solution vector related to background_dof in DEM.
I did not add a new loop, just copy pasted the `evaluate_vector_field` and modified `DataPostprocessorInputs::Vector<dim> &input_data` to `DataPostprocessorInputs::Scalar<dim> &input_data` argument.